libraries/pose/README.md

```diff
@@ -254,72 +274,73 @@ const prediction = await model.predictTopK(posenetOutput, maxPredictions);
 
 ### Webcam
 
-You can optionally use a webcam utility that comes with the library, or spin up your own webcam. This method exists on the `tmPose` module.
+You can optionally use a webcam class that comes with the library, or spin up your own webcam. This class exists on the `tmImage` module.
```

Comment:
I believe you mean to say the `tmPose` module here as it appears that the webcam class is included in pose locally.